man/default_undesirable_functions.Rd

```diff
@@ -10,11 +10,11 @@
 \format{
 A named list of character strings.
 
+An object of class \code{list} of length 21.
 
+An object of class \code{list} of length 3.
 
-
-
-
+An object of class \code{list} of length 3.
```

Comment:
Any idea why these reappeared?

Reply:
Oh because they had `@format` which only suppressed the format due to a bug in roxygen2. I fixed them by setting `@format NULL`